vendor/git-repo-info.js

```diff
@@ -303,7 +306,14 @@ function getCommitData(gitPath, sha) {
   var objectPath = path.join(gitPath, 'objects', sha.slice(0, 2), sha.slice(2));
 
   if (zlib.inflateSync && fs.existsSync(objectPath)) {
-    var objectContents = zlib.inflateSync(fs.readFileSync(objectPath)).toString();
+    let objectContents = zlib.inflateSync(fs.readFileSync(objectPath)).toString();
+
+    // if there's a gpg signature, we remove it
```

Comment:
Is this tested anywhere?

Reply:
no. We have no coverage at all for this file. Maybe I add coverage only for this bit? In any case, at this point I'm seriously considering removing this dependency altogether: it's just not reliable enough.

Reply:
Yeah, I'd say if we're going to be significantly altering this dependency, and it's not deemed reliable anyway, let's just remove it and roll our own.